src/app/core/main.nim

```diff
@@ -4,28 +4,29 @@ import
   ./fleets/fleet_configuration,
   ./tasks/marathon,
   ./tasks/threadpool,
-  ./signals/signals_manager
+  ./signals/signals_manager,
+  ./notifications/notifications_manager
 
 export eventemitter
-export marathon, task_runner, signals_manager, fleet_configuration
+export marathon, task_runner, signals_manager, fleet_configuration, notifications_manager
 
 type StatusFoundation* = ref object
   events*: EventEmitter
   fleetConfiguration*: FleetConfiguration
   threadpool*: ThreadPool
   signalsManager*: SignalsManager
+  notificationsManager*: NotificationsManager
 
 proc newStatusFoundation*(fleetConfig: string): StatusFoundation =
   result = StatusFoundation()
   result.events = createEventEmitter()
   result.fleetConfiguration = newFleetConfiguration(fleetConfig)
   result.threadpool = newThreadPool()
   result.signalsManager = newSignalsManager(result.events)
+  result.notificationsManager = newNotificationsManager(result.events)
```

Comment:
Is this done before or after login? I'm asking because you removed the `onLoggedIn` code I had put in the service. 

If it is done before login, I've had some race conditions on Windows where the setup would be done too soon. I'll test it later to see if it still happens with your PR.